vendor.conf

```diff
@@ -22,7 +22,7 @@ github.com/docker/go-events 9461782956ad83b30282bf90e31fa6a70c255ba9
 github.com/docker/go-metrics d466d4f6fd960e01820085bd7e1a24426ee7ef18
 github.com/docker/go-units 47565b4f722fb6ceae66b95f853feed578a4a51c # v0.3.3
 github.com/docker/libtrust 9cbd2a1374f46905c68a4eb3694a130610adc62a
-github.com/docker/licensing 369e530
+github.com/docker/licensing 5c4c7b4
```

Comment:
please use full commit ID even if for some reason it wasn't a full ID last time.

Reply:
Will do on the next bump.